dts/scenes/StageManager.d.ts

```diff
@@ -42,8 +42,10 @@ export default class StageManager {
      * @param {string} sceneID ID of Scene to transition to
      */
     changeScene: (newScene: string) => void;
-    captionsMuted: boolean;
-    pause: boolean;
+    get captionsMuted(): boolean;

```

Comment:
Is this some new preference in TypeScript that turns these into getter and setter types?